sdk/keyvault/keyvault-certificates/src/transformations.ts

```diff
@@ -279,9 +285,21 @@ export function getDeletedCertificateFromItem(item: DeletedCertificateItem): Del
     createdOn: attributes.created,
     updatedOn: attributes.updated,
     expiresOn: attributes.expires,
-    ...parsedId,
-    ...item,
-    ...item.attributes
+
+    sourceId: parsedId.sourceId,
```

Comment:
It feels weird to spell all these properties out when you've typed the object itself as `any` and erased any benefit given from being explicit here.

Reply:
thank you so much! you've found a bug. I'll update this